docs/manual/en/introduction/Drawing-lines.html

```diff
@@ -10,28 +10,35 @@
 		<h1>[name]</h1>
 		<div>
 			<p>
-				Let's say you want to draw a line or a circle, not a wireframe [page:Mesh].
-				First we need to set up the [page:WebGLRenderer renderer], [page:Scene scene] and camera (see the Creating a scene page).
+				When creating graphics with three.js, sometimes you might want to draw simple shapes like lines or circles, rather than wireframe [page:Mesh].
```

Comment:
TBH, I don't think that the suggest rephrasing makes the text actually "better". It's just different but that does not justify a change.

Do you mind just focusing on just fixing typos?

Reply:
Indeed... In fact, I prefer the original wording.

Reply:
I understand, thank you for your comments. I will just focus on the typos and not try to reinvent the wheel where it is not needed.